terraform/modules/aws-dns-records/resources.route53.tf

```diff
@@ -28,9 +28,20 @@ resource "aws_route53_record" "spf" {
   ttl     = "60"
   records = [
     join(" ", concat(
-      [ "v=spf1" ],
-      [ for ip in var.spf_record_ips : "ip4:${ ip }" ],
-      [ "-all" ]
+      ["v=spf1"],
+      [for ip in var.spf_record_ips : "ip4:${ip}"],
+      ["-all"]
     ))
   ]
 }
+
+resource "aws_route53_record" "srv-server" {
+  for_each = var.srvs
+
+  zone_id = data.aws_route53_zone.rz.zone_id
+  name    = join(".", concat([each.key], local.name_suffix))
+  type    = "SRV"
+  ttl     = "60"
+
+  records = [for t in each.value : join(".", concat([t], local.name_suffix))]
```

Comment:
note: that one gave me quick headache. clever, tho, but a bit confusing too (when looking at the example). Would some interpolation syntax make it more obvious?

Reply:
interpolation would probably be better. the `join(concat` is confusing. But currently local.name_suffix is a list. We could refactor this though to become `[for t in each.value : "${t}.${local.domain_suffix}`. I'll make a follow-up PR for that.